Tests/SwiftStdlibTests/DictionaryExtensionsTests.swift

```diff
@@ -26,6 +26,17 @@ final class DictionaryExtensionsTests: XCTestCase {
         XCTAssertFalse(dict.keys.contains("key2"))
     }
 
+    func testRemoveElementForRandomKey() {
+        var emptyDict = [String: String]()
+        emptyDict.removeValueForRandomKey()
```

Comment:
Do you want to use `XCTAssertNil`, like you did for `RangeReplaceableCollectionExtTests`?